charts/mta-v7.x/README.md

```diff
@@ -14,6 +17,89 @@ The following table lists the configurable parameters of the Mta chart and their
+helm install mta orchestrator-workflows/mta-v7 -n ${TARGET_NS}
+```
+
+### Post-installation
+#### Set up the MTA instance with a Jira Connection
+Define a Jira instance in MTA and establish a connection to it, by following the [Creating and configuring a Jira connection](https://access.redhat.com/documentation/en-us/migration_toolkit_for_applications/7.0/html/user_interface_guide/creating-configuring-jira-connection#doc-wrapper) guide.
+
+#### Edit the `${WORKFLOW_NAME}-creds` Secret
+The token for sending notifications from the MTA-v7.x workflow to RHDH notifications service needs to be provided to the workflow.
+
+Edit the secret `${WORKFLOW_NAME}-creds` and set the value of `NOTIFICATIONS_BEARER_TOKEN`:
+```
+WORKFLOW_NAME=mta-analysis-v7
+oc -n ${TARGET_NS} patch secret "${WORKFLOW_NAME}-creds" --type merge -p '{"data": { "NOTIFICATIONS_BEARER_TOKEN": "'$(oc get secrets -n rhdh-operator backstage-backend-auth-secret -o go-template='{{ .data.BACKEND_SECRET  }}')'"}}'
+```
+
+This secret is used in the `sonataflow` CR to inject the token as an environment variable that will be used by the workflow.
+
+Once the secret is updated, to have it applied, the pod shall be restarted. 
+Note that the modification of the secret does not currently restart the pod, the action shall be performed manually or, if you are following the next section, any change to the sonataflow CR will restart the pod.
+
+Note that if you run the `helm upgrade` command, the values of the secret are reseted.
+
+#### Edit the `mta-analysis-v7` Sontaflow CR:
+
+There is one variable required to be set in the `mta-analysis-v7-props` ConfigMap:
+* **mta.url** - The URL to the MTA application
+* **quarkus.rest-client.notifications.url** - The URL to the backstage application.
+
+We will not set those values directly as quarkus will first try to expand the value of `MTA_URL` and `BACKSTAGE_NOTIFICATIONS_URL` to set the values. Instead, we will set the environment variable in the `mta-analysis-v7` Sontaflow CR:
+
+The value for `BACKSTAGE_NOTIFICATIONS_URL` in the command below is using the current default value, if the name of the backstage deployment or its namespace does not match, please update the value with the correct value from your cluster.
+
+**Please note** that it may take several minutes for the MTA Operator to become available and for the route to be reachable.
+```console
+while [[ $retry_count -lt 5 ]]; do
+    oc -n openshift-mta get route mta && break || sleep 60
+    retry_count=$((retry_count + 1))
+done
+BACKSTAGE_NOTIFICATIONS_URL=http://backstage-backstage.rhdh-operator
+MTA_ROUTE=$(oc -n openshift-mta get route mta -o yaml | yq -r .spec.host)
+oc -n ${TARGET_NS} patch sonataflow mta-analysis-v7 --type merge -p '{"spec": { "podTemplate": { "container": { "env": [{"name": "BACKSTAGE_NOTIFICATIONS_URL": "'${BACKSTAGE_NOTIFICATIONS_URL}'"}, {"name": "MTA_URL", "value": "https://'${MTA_ROUTE}'"}]}}}}'
```

Comment:
Same typo, should be `{"name": "BACKSTAGE_NOTIFICATIONS_URL",  "value": "'${BACKSTAGE_NOTIFICATIONS_URL}'"}`

Reply:
Thx, the fix is part of https://github.com/parodos-dev/serverless-workflows-config/pull/846